pyrdp/mitm/FakeServer.py

```diff
@@ -0,0 +1,274 @@
+#
+# This file is part of the PyRDP project.
+# Copyright (C) 2022
+# Licensed under the GPLv3 or later.
+#
+import os, random, shutil, socket, subprocess, threading, time
+from tkinter import *
+from PIL import Image, ImageTk
+from pyvirtualdisplay import Display
+from pyrdp.logging import log  # TODO: this logs to GLOBAL and not to the session, setup proper logging
```

Comment:
Let's make sure we do per-session logging with JSON events being generated properly as well.

Reply:
Do you see a better way to add per session logging, other that to introduce logging to `state.py`? If yes, that might make more sense. I'm also not entirely sure if we should import the fake server in `state.py` as it is right now or better in `RDPMITM.py`

> with JSON events being generated properly as well.

Do you mean that the logging messages should be parametrized? If yes, I think that should be solved now.

Reply:
Yes, this looks better now. Thanks!